.gitignore

```diff
@@ -0,0 +1,7 @@
+# Pesde packages
+*_packages/
+pesde.lock
+
+# Generated scripts
+.pesde/*
+!.pesde/.gitkeep

```

Comment:
Why?

Reply:
We exclude the subprojects in the `.pesde` directory, but keep the directory. I plan on placing a README there to explain what happens to the directory.